src/lib.rs

```diff
@@ -1236,4 +1262,24 @@ mod tests {
             }],
         );
     }
+
+    #[test]
+    fn regression3() {
```

Comment:
Maybe name the test for the property it's checking?

Reply:
Sure, updated the PR